sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ImplementIDisposableCorrectly.cs

```diff
@@ -103,7 +115,7 @@ public virtual void Dispose()
     public class WithFinalizer : IDisposable // Noncompliant
     {
         public void Dispose()
-//                  ^^^^^^^ Secondary {{'WithFinalizer.Dispose()' should only invoke 'Dispose(true)' and 'GC.SuppressFinalize(this)'.}}
+//                  ^^^^^^^ Secondary {{'WithFinalizer.Dispose()' should also call 'GC.SuppressFinalize(this)'.}}
         {
             Dispose(true);
         }
```

Comment:
please also add the snippet from the ticket (if not already present) with the arrow expression syntax
`public void Dispose() => Dispose(true);`

Reply:
You are right, there was no `=> Dispose(true)`.
And because there are several Dispose methods containing only `Dispose(true);`, let's convert one of them with this trendy arrow expression syntax.